lib/packetfu/protos/hsrp.rb

```diff
@@ -52,11 +52,10 @@ def self.can_parse?(str)
       return false unless UDPPacket.can_parse? str
       temp_packet = UDPPacket.new
       temp_packet.read(str)
-      if temp_packet.ip_ttl == 1 and [temp_packet.udp_sport,temp_packet.udp_dport] == [1985,1985] 
-        return true
-      else 
-        return false
-      end
+      return false unless temp_packet.eth_daddr == '01:00:5e:00:00:02' 
+      return false unless temp_packet.ip_daddr == '224.0.0.2' 
```

Comment:
Ok, I'm on board with the multi-cast address, we should probably amend the layer 2 address though.

As for HSRPv2 support, we should probably treat that as a separate entity, so it can be parsed independently based on different criteria.  That's not necessarily a requirement for this PR, but me just thinking out loud.

Also, FWIW, PacketFu is not necessarily attempting to be an all-in-wonder protocol parsing library, it pitches itself as a mid-level lib and I'm raising this because our support outside of your core stuff is often handled in higher-level stacks that people develop on their own (on top of PacketFu).  This is sort of a core distinction, but we are still happy to have PRs and updates to make it work for use-cases that pre-exist in the library. I also say this because I know absolutely nothing about HSRP, so I defer to your good judgement on parsing requirements and lend my expertise just to prevent false matching.